Description: now manually need to link with libgl
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -36,8 +36,8 @@
 endif
 
 ifeq ($(PLATFORM), LINUX)
-	LDFLAGS += -L/usr/lib/ -I./ $(shell $(PKG_CONFIG) --libs glu sdl2 fmodex)
-	CFLAGS += $(shell $(PKG_CONFIG) --cflags glu sdl2 fmodex)
+	LDFLAGS += -L/usr/lib/ -I./ $(shell $(PKG_CONFIG) --libs glu gl sdl2 fmodex)
+	CFLAGS += $(shell $(PKG_CONFIG) --cflags glu gl sdl2 fmodex)
 endif
 
 #variables d'instalation
